docs/data/joy/components/accordion/accordion.md

```diff
@@ -0,0 +1,61 @@
+---
+product: joy-ui
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">Accordions let users show and hide sections of related content on a page.</p>
+
+:::info
+💡 The Joy UI Accordion component is still in development.
+If you're in need of it, please upvote [**this issue**](https://github.com/mui/material-ui/issues/36281) to help us prioritize the next batch of new components.
```

Comment:
```suggestion
💡 The Joy UI Accordion component is still in development.

If you're in need of it, please upvote [**this issue**](https://github.com/mui/material-ui/issues/36281) to help us prioritize the next batch of new components.

Or you can use the Accordion component from headless libraries and leverage the Joy UI theme. See [examples](integration-with-headless-ui-libraries) below.
```

How about this? @danilo-leal @oliviertassinari @samuelsycamore

Reply:
I've done some tweaking exactly on this part! I think it works having the first two sentences in an info callout. Let me know what you think! 👉 https://deploy-preview-36279--material-ui.netlify.app/joy-ui/react-accordion/